### ISSUE TITLE
Add TeX Gyre Schola

### ISSUE THREAD

**Issue by KyleAMathews** [google/fonts#324]
https://www.fontsquirrel.com/fonts/TeX-Gyre-Schola

Google Fonts doesn't have any Century-style fonts atm afaik.

**Comment by davelab6** [comment]
The GUST fonts are not available under the SIL Open Font License from GUST. It may be possible under the LPPL to sublicense them under OFL, but I've postponed this until upstream do it :)

**Comment by davelab6** [comment]
@twardoch worked with URW to get these under OFL :)

**Comment by twardoch** [comment]
To be more specific, the URW++ Core 35 version 2.0 fonts are now under OFL: 
https://github.com/URWPP/Core_35
The "TeX Gyre Schola" equivalent is **C059**. 

The TeX Gyre Schola (which is under LPPL only for now) has a larger glyphset, some of the glyphs are quite poorly drawn, and the GUST group is working on incorporating the URW++ version 2.0 changes into the TeX Gyre fonts. I hope that once this is done, GUST will release the TeX Gyre fonts under LPPL and under OFL. I will work hard to convince them.

**Comment by Theome** [comment]
Does anyone know where the Core 35 fonts are offered for download under the OFL? I'd like to use some of them in a commercial project, but I cannot find a source for the actual release under that license. 

All references I could find (for example [this article](https://en.wikipedia.org/wiki/Nimbus_Mono_L) about Nimbus Mono L on Wikipedia) point to dead links. It looks like since the acquisition of URW++ all these sources and websites have been deleted:

https://github.com/URWPP/
https://github.com/URWTypeFoundry/
https://urwtype.com
https://urwpp.de

**Comment by thlinard** [comment]
Most recently discussed in https://github.com/google/fonts/issues/288#issuecomment-2869115735

**Comment by aaronbell** [comment]
@davelab6 Can you review and see if Adam's comments are sufficient to resolve the GUST/OFL license issue?
